src/components/QuranReader/index.tsx

```diff
@@ -55,9 +56,10 @@ const QuranReader = ({
     selectTranslations,
   ) as TranslationsSettings;
   const { selectedTafsirs, isUsingDefaultTafsirs } = useSelector(selectTafsirs) as TafsirsSettings;
+  const reciter = useSelector(selectReciter, shallowEqual);
   const { data, size, setSize, isValidating } = useSWRInfinite(
     (index) =>
-      getRequestKey(
+      getRequestKey({
```

Comment:
Me and @tifa2UP discussed this recently. There is no organizational benefit of converting this into an object vs having multiple params both ways `getRequestKey` is still bloated.

Reply:
Chatted with Mujajir offline. The benefits are making it an object is that: 

1. The order of the parameter doesn't matter
2. you don't have to explicitly pass nulls.

I think that this makes more sense. Let's leave it that way @osamasayed